src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx

```diff
@@ -4,6 +4,11 @@ tags:
   - Mobile monitoring
   - API guides
 metaDescription: 'Customize your mobile agent instrumentation to send custom data and metrics to New Relic.'
+redirects:
+  - /node/13986
+  - /node/13991
```

Comment:
@nwhite222 What are these for?

Reply:
I'm not sure...I think those were added automatically. I can see if anything happens when I delete them.